pkg/control_room/control_center.go

```diff
@@ -0,0 +1,99 @@
+package control_room
+
+import (
+	"container/heap"
+
+	"github.com/TNSEngineerEdition/WailsClient/pkg/city"
+	"github.com/umahmood/haversine"
+)
+
+type ControlCenter struct {
+	city         *city.City
+	cachedRoutes map[[2]uint64][]*city.GraphNode
+}
+
+func CreateControlCenter(cityData *city.City) ControlCenter {
+	c := ControlCenter{
+		city:         cityData,
+		cachedRoutes: make(map[[2]uint64][]*city.GraphNode),
+	}
+
+	tramTrips := cityData.GetTramTrips()
+	for _, tripData := range tramTrips {
+		for idx := 0; idx < len(tripData.Stops)-1; idx++ {
+			firstStop := tripData.Stops[idx]
+			secondStop := tripData.Stops[idx+1]
```

Comment:
I believe Go supports the tuple syntax:

```suggestion
			firstStop, secondStop := tripData.Stops[idx], tripData.Stops[idx+1]
```